docs/site/.helm/templates/15-modules-watcher.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: registry-modules-watcher
+  annotations:
+    "werf.io/replicas-on-creation": "1"
+  labels:
+    app: registry-modules-watcher
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: registry-modules-watcher
+  template:
+    metadata:
+      labels:
+        app: registry-modules-watcher
+      annotations:
+        checksum/auths: {{ .Values.global.moduleWatcherAuths | sha256sum }}
+    spec:
+      imagePullSecrets:
+        - name: deckhouse-registry-site
+      priorityClassName: {{ pluck .Values.werf.env .Values.priorityClassName | first | default .Values.priorityClassName._default }}
+      serviceAccountName: modules-watcher
+      containers:
+        - name: registry-modules-watcher
+          command:
+            - /app/registry-modules-watcher
+            - --watch-registries={{ (pluck .Values.werf.env .Values.moduleWatcher.registries | first | default .Values.moduleWatcher.registries._default) | join "," }}
+            - --scan-interval={{ pluck .Values.werf.env .Values.moduleWatcher.scanInterval | first | default .Values.moduleWatcher.scanInterval._default }}
+            - --v=1
```

Comment:
I think we have to change this, to `--v=4` for production

Reply:
Done

Reply:
Fixed